### PR TITLE
hinode.pics - Ads etc...

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [x] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 https://hinode.pics/calendars

### Add your comment and screenshots

<details><summary>screenshot</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: JapaneseFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -101,6 +101,7 @@ kijonotakuhaibin.com#?#.plugin-memo:has(> div.side > div[id^="side-koukoku"])
 domani.shogakukan.co.jp#?#.textwidget:has(> div[id^="div-gpt-ad"])
 kijomatomelog.com#?#.sub_right > .sidewrapper:has(> .sidetitlebody > .sidetitle:contains(スポンサードリンク))
 asahi.com#?#.p-kurashi__list > div.p-kurashi__listItem:has(> div[id^="div-gpt-ad-"])
+hinode.pics#?##right_wrap > div.right_plugin_wrap:contains(広告)
```
Comment (zloyden) [inline JapaneseFilter/sections/css_extended.txt:104]:
```
hinode.pics#?##right_wrap > div.right_plugin_wrap:has(> div.right_plugin_title:contains(広告))
```
You should specify the element which contains it.
However, I'd prefer to use the rule without `:contains()`.
```
hinode.pics#?##right_wrap > div.right_plugin_wrap:has(> div.right_plugin_body > div[style] > ins.adsbygoogle)
```

File: SocialFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -17,6 +17,7 @@ quantamagazine.org#?#.post__sidebar__content > div.post__sidebar__content__inner
 mechacomic.jp#?##main > section.c-content > div.area_btn_sns:upward(1)
 corp.helpfeel.com#?#body > .rebranding__section--last > .w-container:has(> .rebranding__share__buttons)
 newsyou.info#?#nofollow > a[href^="https://t.me/joinchat"]:upward(1)
+hinode.pics#?##right_wrap > div.right_plugin_wrap:contains(ソーシャルボタン)
```
Comment (zloyden) [inline SocialFilter/sections/css_extended.txt:20]: You can create a rule without using `:contains()`.
```
hinode.pics#?##right_wrap > div.right_plugin_wrap:has(> div.right_plugin_body > div#right_plugin_social_buttons_wrap)
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/css_extended.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/css_extended.txt]: Thank you

File: SocialFilter/sections/css_extended.txt (hunk 2 above)
Comment (SKEIDs) [reply SocialFilter/sections/css_extended.txt]: Thank you

**Review by zloyden — APPROVED** [review]
You are welcome :)

**Comment by zloyden** [comment]
> fix bad rules

I woudn't say that they are bad. `:contains()` is useful in some cases, but here we can avoid using it. That's all :)

**Pull request merged**